src/shared/components/Animations/ListItem.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import './list-item.less';
+
+export interface ListItemProps {
+  label: string;
+  projectNumber?: number;
+  description?: string;
+  logo?: string;
+  deprecated?: boolean;
+  onClick?(): void;
+  onEdit?(): void;
```

Comment:
Instead of having `onEdit`, can we think of a more generic way of displaying a menu on an item. That menu could be a simple button (passed as a prop) or a more elaborate one?

Also maybe we should add an `onHover` prop as well. My thinking is that we may not check the permissions of every single items at once, but only `onHover`?

Reply:
I like the onHover Idea, we can use it in many places, I'll think about adding it separately as a custom hook or something and then adding it afterwards

Reply:
removed `onEdit`, I think its better for the parent component to simply pass in whatever it needs there